nexum_core/BENCHMARKS.md

```
@@ -0,0 +1,226 @@
+### CI Workflow
+```yaml
+benchmarks:
+  name: Rust benchmarks
+  runs-on: ubuntu-latest
+  if: github.event_name == 'pull_request'
+  # ... runs benchmarks and compares results
+```
+
+## Interpreting Results
+
+### Criterion Output
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)